pkg/mei/mei.go

```diff
@@ -0,0 +1,89 @@
+package mei
+
+import (
+	"encoding/binary"
+	"fmt"
+	"os"
+	"syscall"
+	"unsafe"
+
+	ioctl "github.com/vtolstov/go-ioctl"
+)
+
+// DefaultMEIDevicePath is the path of the default MEI device. This file will be
+// present if the "mei_me" kernel module is loaded.
+var DefaultMEIDevicePath = "/dev/mei0"
+
+// HECIGuids maps the known HECI GUIDs to their values. The MEI interface wants
+// little-endian. See all the GUIDs at
+// https://github.com/intel/lms/blob/master/MEIClient/Include/HECI_if.h
+var (
+	// "8e6a6715-9abc-4043-88ef-9e39c6f63e0f"
+	MKHIGuid = ClientGUID{0x15, 0x67, 0x6a, 0x8e, 0xbc, 0x9a, 0x43, 0x40, 0x88, 0xef, 0x9e, 0x39, 0xc6, 0xf6, 0x3e, 0xf}
+)
+
+// see include/uapi/linux/mei.h
+var (
+	IoctlMEIConnectClient = ioctl.IOWR('H', 0x01, uintptr(len(ClientGUID{})))
```

Comment:
Is there any way to exclude this new dependency?

Reply:
*named constants, lol

Reply:
it's a super-small dependency that exposes Linux's `IOWR`. I can reimplement it in this package as a small function if that's OK

Reply:
done, more changes incoming

Reply:
since it's super small, i'm actually okay with it. but we should also go replace pkg/ipmi's IOWR function and use this new dependency.